assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -45,7 +47,30 @@ export default {
         "header-chat-link",
         "chat:rerender-header"
       );
+
       api.addToHeaderIcons("header-chat-link");
+
+      api.decorateCookedElement((elem) => {
+        const chatTranscriptElements = elem.querySelectorAll(
+          ".discourse-chat-transcript"
+        );
+        chatTranscriptElements.forEach((el) => {
+          const dateTimeRaw = el.dataset["datetime"];
+          const dateTimeLinkEl = el.querySelector(
+            ".chat-transcript-datetime a"
+          );
+
+          if (currentUserTimezone) {
+            dateTimeLinkEl.innerText = moment
+              .tz(dateTimeRaw, currentUserTimezone)
+              .format(I18n.t("dates.long_no_year"));
+          } else {
```

Comment:
Is this in the acceptance test? Should be easy to include.

Reply:
Yep done!